sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -188,6 +265,93 @@ To run side-effects like logging or analytics whenever some specific values chan
 <p>{count} doubled is {doubled}</p>
 ```
 
+> Use `$effect` with caution - most of the time, you're better off using a different pattern. Most importantly, writing to `$state` within an `$effect` is almost always an anti-pattern, and when not careful could result in the `$effect` rerunning over and over again. Below are some use cases and what to use instead.
```

Comment:
Something like?

"`$effect` was designed to be a safe place for side-effects. If you're managing state or dataflow, you should use it with caution – most of the time, you're better off using a different pattern"

Reply:
```suggestion
> `$effect` was designed for managing side effects such as logging or connecting to external systems like third party libraries that have an imperative API. If you're managing state or dataflow, you should use it with caution – most of the time, you're better off using a different pattern. Below are some use cases and what to use instead.
```